spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/ldap/embedded/EmbeddedLdapProperties.java

```diff
@@ -109,4 +123,34 @@ public void setPassword(String password) {
 
 	}
 
+	public static class Validation {
+
+		/**
+		 * Enable LDAP schema validation
+		 */
+		private boolean enabled = true;
```

Comment:
What is the semantic of that flag? Does it make sense to set a schema and disable validation? Why do you need to set both a schema and the validation flag. Perhaps we should check for the presence of the schema only?